.yarn/versions/2e696b36.yml

```diff
@@ -0,0 +1,30 @@
+releases:
+  "@yarnpkg/builder": patch
```

Comment:
Only need a patch release on core and cli, the rest can be declined

Reply:
Ah cool, gotcha! Thanks 👍